### DISCUSSION TITLE
Vertical Taskbar

### DISCUSSION THREAD

**Discussion by jsuelwald** [valinet/ExplorerPatcher#192] [Ideas]
Is it possible to have the taksbar aranged vertically like in Win10?

**Comment by Gaurav-Original-ClassicShellTester** [comment]
Yes. Once you get the classic taskbar working by installing ExplorerPatcher's DLL, unlock it by right clicking. Drag it to whatever edge and lock it again.

**Reply by bhagwad** [reply]
Holy cow, this works!
